arm-servicebus/2015-08-01/swagger/servicebus.json

```diff
@@ -41,7 +39,7 @@
         "tags": [
           "Namespaces"
         ],
-        "operationId": "Namespaces_ListBySubscription",
+        "operationId": "Namespaces_List",
```

Comment:
@vishrutshah we can definitely move these changes to a new swagger version. 
Do you want us to create a new PR as well?